### PR TITLE
Menu/resume

### PR DESCRIPTION
**Pull request by QbaF**
As divided for three types described separatelly, I propose that small menu/resume.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/dashboards/header-footer.markdown [hunk 1]
```diff
@@ -9,6 +9,11 @@ Some dashboard cards have support for header and footer widgets. These widgets f
 Screenshot of an entities card with a picture header and buttons footer.
 </p>
 
+Could be:
+- [picture] /dashboards/header-footer/#picture-header--footer
+- [buttons] /dashboards/header-footer/#buttons-header--footer or
+- [graph] /dashboards/header-footer/#graph-header--footer
```
Comment (frenck) [inline source/dashboards/header-footer.markdown:15]: - These are on this page? Show in the sidebar?
- This doesn't work?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/dashboards/header-footer.markdown (hunk 1 above)
Comment (QbaF) [reply source/dashboards/header-footer.markdown:15]: Are on this page. They are just 3 types of header/footer.

Comment (frenck) [reply source/dashboards/header-footer.markdown:15]: Ok, so I'm confused on why this is added in that case?

(besides the fact, it doesn't work..)

Comment (QbaF) [reply source/dashboards/header-footer.markdown:15]: Ok. When I was looking in the documentation for info about footers/headers I found it firstly not clear (mainly why configuration variable `type` has only `picture` argument). Just for a few seconds but I thought that some users could find it useful when supplemented with a short menu at the beginning. And I thought that I can somehow help that great community with that small contribution when proposing some changes. 

You can use it, correct it or point me out what or how I should correct, it or throw it away.  I will not feel offended, I promise;) Maybe completely fresh to GitHub, as I am, shouldn't try to cooperate with changes proposals as maybe it brings more troubles than goods.

This was the answer to why I added those links on the page.

Why these links don't work - I have no idea. I wrote it based on other parts of the code while checking links on the page. Conclusion could be as above.

**Comment by frenck** [comment]
Going to close this PR and friendly decline its contents. Besides the case is not clear to me, it also isn't working either.

**Pull request closed without merging**